en/docs/learn/consume-api/account-management-dev-portal/advanced-topics/adding-a-user-signup-workflow.md

```diff
@@ -0,0 +1,51 @@
+# Adding a User Signup Workflow
+
+## Engaging the Approval Workflow Executor in API Manager
+
+1. Sign in to the API-M Management Console.
+
+     `https://<Server-Host>:9443/carbon`
+
+     `https://localhost:9443/carbon`
+
+2. Select **Browse** under **Resources**.
+
+    ![Browse resources]({{base_path}}/assets/img/learn/wf-extensions-browse.png)
+
+3.  Go to `/_system/governance/apimgt/applicationdata/workflow-extensions.xml` resource, disable the **UserSignUpSimpleWorkflowExecutor** and enable **UserSignUpApprovalWorkflowExecutor** for user self sign up.
+
+    ```
+    <WorkFlowExtensions>
+        ...
+            <!--UserSignUp executor="org.wso2.carbon.apimgt.impl.workflow.UserSignUpSimpleWorkflowExecutor"/-->
+            <UserSignUp executor="org.wso2.carbon.apimgt.impl.workflow.UserSignUpApprovalWorkflowExecutor"/>
+        ...
+    </WorkFlowExtensions>
+    ```
+
+4.  Navigate to the Developer Portal. 
+
```

Comment:
extra lines

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/1555/commits/dcc2e157314cf300788569c01c2ad89f0877c4b3